accessibility/README.md

```diff
@@ -8,26 +8,30 @@ Web accessibility, [according to the W3C](https://www.w3.org/WAI/intro/accessibi
+An accessible site should also work well with [assistive technology](https://www.atia.org/at-resources/what-is-at/) such as screen readers, dictation software, magnification software, braille terminals and alternative input devices. An accessibile site may conform to accessibility standards such as W3C's [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG20/) (WCAG). However an accessible site is not necessarily WCAG-compliant, and a WCAG-compliant site is not necessarily accessible.
 
-Accessibility can also mean ensuring that content and online services are available regardless of users' technology or network quality. While this can be hard to achieve for complex sites, it makes good sense to build sites so they include as many people as possible. It is not good to exclude users who do not have the most modern browsers, or who have older computers, or are on inconsistent networks.
+Accessibility can also mean ensuring that content and online services are available regardless of users' technology or network quality. While this can be hard to achieve for complex sites, it makes good sense to build sites so they enable access for as many people as possible. It is not good to exclude users who do not have the most modern browsers, or who have older computers, or are on inconsistent networks.
 
 ## Why is it important?
 
-Disability can take many forms, and affect people in many different ways. With more and more information and services moving online, and an estimated [20% of the UK population having a disability](https://www.gov.uk/government/statistics/disability-facts-and-figures) is is inevitable that the accessibility of most Web sites will increasingly matter to users.
+Disability can take many forms, and affect people in many different ways. With more and more information and services moving online, and an estimated [20% of the UK population having a disability](https://www.gov.uk/government/statistics/disability-facts-and-figures), is is inevitable that the accessibility of most Web sites will increasingly matter to users.
 
-While it may be easy to dismiss Web accessibility as not important, or even worse dismiss people with disabilities as not important, it 
+While it may be easy to dismiss Web accessibility as not important, or even worse dismiss people with disabilities as not important, it is increasingly obvious to many that ensuring all people have the opportunity to fully participate in all aspects of life is morally right and societally beneficial.
 
 ## What's happening with it?
```

Comment:
It would be good if we could also briefly mention the UK Equality Act 2010 in the context of pre-existing legislation that affects accessibility, including introducing the concept of 'reasonable adjustments'.